### PR TITLE
Wps hook

### PR DESCRIPTION
**Pull request by huard**
A few small changes and fixes in the process to create a WPS Process based on xclim classes. Master version of finch has now one WPS process based on xclim. 

I've started modifying the tg_mean class according to Lars' specs, hence the name change.

### CODE REVIEW

File: xclim/temperature.py [hunk 1]
```diff
@@ -47,13 +47,13 @@ def cfprobe(self, da):
...
-              long_name="{freq} mean temperature",
-              standard_name="{freq} mean temperature",
-              description="{freq} of daily mean temperature.",
-              keywords='',
-              compute=_ind.tg_mean,)
+tmmean = Tas(identifier='tmmean',
+             units='K',
+             long_name="Mean daily mean temperature",
+             standard_name="air_temperature",
+             description="{freq} of daily mean temperature.",
```
Comment (tlogan2000) [inline xclim/temperature.py]: Will '{freq}' be translated to something more verbose using a dictionary?  Will need to apply both the freq and the method no? 'YS' yields annual but need to specify that used used .mean(dim='time') somehow so the description reads: 'Annual mean of daily mean temp' or equivalent

Comment (huard) [reply xclim/temperature.py]: Typo on my end

Comment (huard) [reply xclim/temperature.py]: Yes, see _attrs_mapping in utils.UnivariateIndicator.

Comment (tlogan2000) [reply xclim/temperature.py]: dacc

**Pull request merged**